pkg/config/config.go

```diff
@@ -124,47 +126,68 @@ func (c *Config) ValidateAndCompleteConfig() error {
 		}
 	}
 
-	if err := c.validatePythonPackagesHaveVersions(); err != nil {
-		return err
+	if len(c.Build.PythonPackages) > 0 && c.Build.PythonRequirements != "" {
+		return fmt.Errorf("Only one of python_packages or python_requirements can be set in your cog.yaml, not both")
 	}
 
-	if c.Build.GPU {
-		if err := c.validateAndCompleteCUDA(); err != nil {
+	// Load python_requirements into memory to simplify reading it multiple times
+	if c.Build.PythonRequirements != "" {
+		b, err := os.ReadFile(path.Join(projectDir, c.Build.PythonRequirements))
```

Comment:
It might be worth considering `bufio.Scanner` here to handle the possibility that someone's checked in a file with CRLF endings.

Reply:
Good idea! Fixed.